code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -149,3 +149,10 @@
 	icon_state = "pill18"
 	list_reagents = list("insulin" = 50)
 	roundstart = 1
+
+/obj/item/weapon/reagent_containers/pill/mining
+	name = "Basic brute and burn pill"
+	desc = "Contains 20 units of bicaridine and kelotane, for a cheap way to mend those lacerations!"
+	icon_state = "pill20"
+	list_reagents = list("Bicaridine" = 20, "Kelotane" = 20)

```

Comment:
Yes, its meant as a simple alternative to the normal medipen. which, mind you has the following regents
	list_reagents = list("salbutamol" = 10, "leporazine" = 15, "tricordrazine" = 15, "epinephrine" = 10, "miningnanites" = 2, "omnizine" = 5)
the normal magic pen will stabilize your body temp, heal you of some toxin, stop you from taking oxyloss, heal oxyloss, and heal you of 100 dmg, all over the course of about... 5 seconds, give or take. all for 500 pts.
this is about 150 pts per pill, but your healing takes way longer, you dont get oxyloss protection, nor temp stabilization, nor toxin healing.